README.md

```diff
@@ -124,11 +124,40 @@ terraform show -json tfplan > output.json
 tf-summarize output.json                      # summary in table format
 ```
 
-#### Github Actions Workflow
+#### GitHub Actions Workflow
 
-Please refer this sample [github actions file](.github/workflows/demo.yml) and the sample runs [here](https://github.com/dineshba/tf-summarize/actions/workflows/demo.yml)
+Please refer this sample [GitHub actions file](.github/workflows/demo.yml) and the sample runs [here](https://github.com/dineshba/tf-summarize/actions/workflows/demo.yml)
 
-> Note: If you are using `hashicorp/setup-terraform` github action to setup terraform, ensure terraform_wrapper is set to false.
+> Note: If you are using `hashicorp/setup-terraform` GitHub action to set up terraform, ensure terraform_wrapper is set to false.
+
+#### Codefresh example
```

Comment:
Could you add this section link in the header section of the page?

(as second example bullet point)
![image](https://user-images.githubusercontent.com/13361187/220359287-d88ec903-6ef9-4220-8e51-a02cad2eda4c.png)

Reply:
I've updated the TOC for this. Looks like we both used the intelli-j insert one